Makefile

```diff
@@ -7,6 +7,9 @@ unit:
 integration:
 	node_modules/.bin/cucumber-js --tags $(INTEGRATIONS_TAGS) tests/cucumber/features --require-module ts-node/register --require tests/cucumber/steps/index.js
 
+display-all-js-steps:
+	tail -n +135 tests/cucumber/steps/steps.js | grep -v '^ *//' | awk "/(Given|Then|When)/,/',/" | grep -E "\'.+\'"  | sed "s/^[^']*'\([^']*\)'.*/\1/g"
```

Comment:
@tzaffi Can you an add inline comment explaining why the command tails a specific file rather than `find`-ing appropriate files?

I leave the question presuming you ran into some kind of parsing issue.  If true, I think it'd help future maintainers understand the deviation + specific configuration.

Reply:
Thanks for the note - Resolving.